autoextract_poet/exceptions.py

```diff
@@ -0,0 +1,9 @@
+from dataclasses import dataclass
```

Comment:
`dataclasses` is not available on Python 3.6 😕

Reply:
I also have mixed feelings here:
- I'd like to ditch Python 3 since this is a new project
- Agree with you that this could represent a barrier when migrating existing spiders
- Although, I believe Python 3.6 spiders could be easily migrated to Python 3.7, as opposed to Python 2 spiders being migrated to Python 3.X

Reply:
@kmike, we're welcome to share your two cents as well 😄

Reply:
There is a backport for Python 3.6, which can be conditionally installed in setup.py.

Reply:
Fixed